src/index.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+const app = require('./app.js');
+
+require('dotenv').config();
+
+const PORT = process.env.PORT || 3005;
+
+app.listen(PORT, () => {});
```

Comment:
Consider adding a log statement in the callback of `app.listen` to confirm the server is running and on which port. For example: `console.log(`Server is running on port ${PORT}`);`